### PR TITLE
Add additional proto and documentation to trade interfaces to match server.

### PR DESCRIPTION
**Pull request by 0xAlcibiades**

### CODE REVIEW

**Comment by 0xAlcibiades** [comment]
Folks, if you would not mind chiming in with any suggested changes to the API reference to make it better, or more complete.

**Review by ghost — APPROVED** [review]
LGTM re:documentation.
I did not review proto files

File: README.md [hunk 1]
```diff
@@ -1,14 +1,765 @@
...
+### Auth service
+
+The Authentication Service in Valorem Trade API enables users to authenticate
+themselves via SIWE, and receive the necessary credentials to access the other
+services provided by the API. The Auth service uses session cookies to store
+authentication information. Auth sessions are backed by
+cryptographically signed cookies. These cookies are generated when they’re
+not found or are otherwise invalid. When a valid, known cookie is received
+in a request, the session is hydrated from this cookie. These cookies validated
+server-side. This provides compatibility with both browser and
```
Comment (dbelv) [inline README.md]:
```suggestion
in a request, the session is hydrated from this cookie. These cookies are validated
server-side. This provides compatibility with both browser and
```

**Review by dbelv — APPROVED** [review]
Minor comment - doesn't stop merge.

**Review by dalechyn — APPROVED** [review]

**Pull request merged**